src/esp/gfx/Simulator.h

```diff
@@ -6,6 +6,7 @@
 
 #include "esp/core/esp.h"
 #include "esp/core/random.h"
+#include "esp/nav/PathFinder.h"
```

Comment:
I don't think this is necessary. There is already a forward declaration below.

Reply:
For `NavMeshSettings`

Reply:
Ah. To be consistent, you may want to consider forward declaring NavMeshSettings.

Reply:
Done.